docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -137,68 +136,146 @@ To output predicted class probabilities instead of predicted classes, you can us
 ```
 
 Here's how to specify a particular model to use for prediction instead of AutoGluon's default model-choice:
 
 ```{.python .input}
 i = 0  # index of model to use
-model_to_use = predictor.model_names[i]
+model_to_use = predictor.get_model_names()[i]
 model_pred = predictor.predict(datapoint, model=model_to_use)
 print("Prediction from %s model: %s" % (model_to_use, model_pred))
 ```
 
-The `predictor` also remembers what metric predictions should be evaluated with, which can be done with ground truth labels as follows:
+We can easily access various information about the trained predictor or a particular model:
+```{.python .input}
+model_to_use = predictor.model_names[i]
+specific_model = predictor._trainer.load_model(model_to_use)
 
+# Objects defined below are dicts of various information (not printed here as they are quite large):
+model_info = specific_model.get_info()
+predictor_information = predictor.info()
 ```
-y_pred = predictor.predict(test_data)
+
+The `predictor` also remembers what metric predictions should be evaluated with, which can be done with ground truth labels as follows:
+
+```{.python .input}
+y_pred = predictor.predict(test_data_nolabel)
 predictor.evaluate_predictions(y_true=y_test, y_pred=y_pred, auxiliary_metrics=True)
 ```
 
 However, you must be careful here as certain metrics require predicted probabilities rather than classes.
-Since the label columns remains in the `val_data` DataFrame, we can instead use the shorthand:
+Since the label columns remains in the `test_data` DataFrame, we can instead use the shorthand:
 
-```
-predictor.evaluate(val_data)
+```{.python .input}
+predictor.evaluate(test_data)
 ```
 
 which will correctly select between `predict()` or `predict_proba()` depending on the evaluation metric.
 
+## Interpretability (feature importance)
+
+To better understand our trained predictor, we can estimate the overall importance of each feature:
+
+```{.python .input}
+importance_scores = predictor.feature_importance(test_data)
+print(importance_scores)
+```
+
+Computed via [*permutation-shuffling*](https://explained.ai/rf-importance/), these feature importance scores quantify the drop in predictive performance (of the already trained predictor) when one column's values are randomly shuffled across rows. The top features in this list contribute most to AutoGluon's accuracy (for predicting when/if a patient will be readmitted to the hospital). Features with non-positive importance score hardly contribute to the predictor's accuracy (at least not on an individual basis, these features may still provide useful predictive signal through interaction-effects with other features). These scores facilitate interpretability of the predictor's global behavior (which features it relies on for *all* predictions) rather than [local explanations](https://christophm.github.io/interpretable-ml-book/taxonomy-of-interpretability-methods.html) that only rationalize one particular prediction.
```

Comment:
negative scoring features generally are purely detrimental/useless, since interaction-effects are impacted by the shuffling.

Reply:
I meant to say that with interaction effects the negative permutation-scores may not properly correlate with remove-and-retrain performance, but it seems too confusing so just removed this statement.